Update tobyfit docs (#1335)
pace-neutrons/Horace: documentation/user_docs/docs/manual/Tobyfit.rst

Tobyfit
#######

.. |SQW| replace:: :math:`S(\mathbf{Q}, \omega)`

Introduction
============

> The purpose of Tobyfit is to enable you to fit to Horace datasets taking into

Reviewer: fit what?

account the broadening of the data arising from the resolution of the
instrument. Even if the excitations in your sample were infinitely long-lives
you would have non-zero broadening in energy and wave vector arising from
numerous contributions.

The spread of energies arise from:

- the non-zero width in the moderator pulse;

- the non-zero pulse width from the Fermi or disk choppers;

- the different flight-times for the neutrons due to different distances that
  neutrons travel depending on the point of emission from the moderator, point

>   of scattering ion the sampel and point of absorption in the detector.

Reviewer: ```suggestion
  of scattering on the sample and point of absorption in the detector.
```

increase in the energy width of peaks (thereby giving an illusory shorter
lifetime of the excitations), they can also shift the positions of peaks
depending on, for example, the curvature of dispersion relation within the
resolution function. This can result in incorrect values of exchange constants
being extracted from the data is resolution is ignored.

Tobyfit uses the :ref:`multifit <manual/Multifit:Multifit>` fitting interface to
enable you fit your data to a model for |SQW| together with background
functions.

.. note::

   By default the |SQW| model is global to all of the datasets you pass to
   Tobyfit, i.e. a single model that applies with the same parameter values to
   all datasets, with independent background functions for each of the
   datasets.

   However, see :ref:`background functions <manual/Multifit:Background
   functions>` for more info.

Tobyfit operates just like :ref:`multifit_sqw <manual/Multifit:multifit_sqw>`,
with the same set of capabilities: controlling parameter setting, binding, etc.
that all other :ref:`multifit variants <manual/Multifit:Multifit>` have. The
difference is that Tobyfit uses instrument information in the sqw objects to

> convolute the |SQW| model(s) with the instrument resolution function using a

Reviewer: ```suggestion
convolve the |SQW| model(s) with the instrument resolution function using a
```

*********************************************

The first thing you need to do is provide the sample and instrument information

> to your datasets. You can do this by creating a sample object. There is an

Reviewer: FOR your datasets???

```suggestion
to your datasets. For samples, you can do this by creating a sample object. There is an
```
